cmd/root.go

```diff
@@ -107,10 +116,18 @@ func startScanner(namespace string, clusterWide bool, scanner Scanner) error {
 
 func init() {
 	rootCmd.Flags().StringP("namespace", "n", "", "namespace to be evaluated")
-	rootCmd.Flags().BoolP("cluster", "c", false, "Scan cluster wide resources")
-	rootCmd.Flags().StringP("kubewarden-namespace", "k", defaultKubewardenNamespace, "namespace where the Kubewarden components (e.g. Policy Server) are installed (required)")
-	rootCmd.Flags().StringP("policy-server-url", "u", "", "Full URL to the PolicyServers, for example https://localhost:3000. Audit scanner will query the needed HTTP path. Useful for out-of-cluster debugging")
+	rootCmd.Flags().BoolP("cluster", "c", false, "scan cluster wide resources")
+	rootCmd.Flags().StringP("kubewarden-namespace", "k", defaultKubewardenNamespace, "namespace where the Kubewarden components (e.g. PolicyServer) are installed (required)")
+	rootCmd.Flags().StringP("policy-server-url", "u", "", "URI to the PolicyServers the Audit Scanner will query. Example: https://localhost:3000. Useful for out-of-cluster debugging")
 	rootCmd.Flags().VarP(&level, "loglevel", "l", fmt.Sprintf("level of the logs. Supported values are: %v", logconfig.SupportedValues))
 	rootCmd.Flags().BoolVarP(&printJSON, "print", "p", false, "print result of scan in JSON to stdout")
-	rootCmd.Flags().StringSliceVarP(&skippedNs, "ignore-namespaces", "i", nil, "Comma separated list of namespace names to be skipped from scan")
+	rootCmd.Flags().StringSliceVarP(&skippedNs, "ignore-namespaces", "i", nil, "comma separated list of namespace names to be skipped from scan. This flag can be repeated")
+	rootCmd.Flags().BoolVar(&insecureSSL, "insecure-ssl", false, "skip SSL cert validation when connecting to PolicyServers endpoints. Useful for development")
+
+	rootCmd.Flags().StringVarP(&caCertFile, "filepath-cacert", "f", defaultPolicyServersCARootFile, "File path to CA cert in PEM format of PolicyServer endpoints (setting KUBEWARDEN_CACERT_PEM_POLICYSERVERS has preference)")
+
+	// Prioritize env var before default value
+	if caEnv, present := os.LookupEnv("KUBEWARDEN_CACERT_PEM_POLICYSERVERS"); present {
```

Comment:
I didn't want to import viper for only this.. so I did it this way.

Reply:
I see, it's a bit strange however that only one of our flags can be set via environment variable. Moreover, it's not something that can be easily discovered: the help output doesn't seem to hint about that, the README doesn't mention it.

Why don't we remove this line of code and add a dedicated issue about integrating viper to allow each flag to be set via env variable too?

Reply:
Dropped env var in https://github.com/kubewarden/audit-scanner/pull/75/commits/51259a95300b95565743c014c9a7396727349fd6.